src/PackageCompiler.jl

```diff
@@ -40,14 +40,12 @@ function julia_cpu_target(::Nothing)
 end
 
 """
-Reverts a forced compilation of the system image.
-This will restore any previously backed up system image files, or
-build a new, clean system image.
+Reverts a forced compilation of the system image. Calls out to 
+`Main.BuildSysImg.build_sysimg(force = true)`.
 """
 function revert(debug = false)
-    syspath = default_sysimg_path(debug)
-    sysimg_backup = dirname(get_backup!(debug))
-    copy_system_image(sysimg_backup, syspath)
+    include(joinpath(Sys.BINDIR, Base.DATAROOTDIR, "julia", "build_sysimg.jl"))
+    Main.BuildSysImg.build_sysimg(force = true)
```

Comment:
I think that functionality is deprecated... Did you actually check that it works and improves the timing?

Reply:
Yeah. This stuff is in the Julia 1.1 docs, but I think it's not going to be in Julia 1.2.

Reply:
See here: https://docs.julialang.org/en/v1/devdocs/sysimg/#Main.BuildSysImg.build_sysimg

Reply:
Are you sure that this solves the startup latency? On first sight, I can't see anything, that `build_sysimg.jl` does better than what PackageCompiler was doing...

Reply:
I'll post a new set of benchmarks after lunch.

Reply:
Actually, I don't think this is the best approach anyway. I ran into a cross-platform bug and GitHub issues suggest... switching to `PackageCompiler,`

I'll try the approach with `PrecompileCommand` or something.

Reply:
`PrecompileCommand ` whats that?

Reply:
https://github.com/JuliaLang/PackageCompiler.jl/blob/fa1f43a56ef7f448a4c9af57dedb37174eaa36f8/src/incremental.jl#L53 this thing, figured it would do what we need (precompile things)

Reply:
That's pretty unrelated ;)

Reply:
rip. Will close this then, until I get a better sense of how the different pieces relate.

Reply:
Thanks for the effort :) I took a look as well, but it's not 100% simple...

Reply:
No problem.